pycheribuild/projects/disk_image.py

```diff
@@ -642,6 +643,21 @@ def make_efi_partition(self, efi_partition: Path):
                 self.run_cmd(mtools_bin / "mdir", "-i", efi_partition, "-/", "::")
                 # self.run_cmd(mtools_bin / "mdu", "-i", efi_partition, "-a", "::")
 
+    def make_rootfs_tarball(self):
+        # write out the manifest file:
+        self.mtree.write(self.manifest_file, pretend=self.config.pretend)
+        try:
+            bsdtar_path = shutil.which(str(self.bsdtar_cmd))
+            if not bsdtar_path:
```

Comment:
Should probably be a fatal error for !pretending?

Reply:
Fair; I just copied this from `cheribsd.py`, which might merit a similar change.